Cargo.toml

```diff
@@ -3,13 +3,13 @@
 
 [workspace]
 members = [
-  "alloc",
   "builder",
   "crashtracker",
   "crashtracker-ffi",
-  "profiling",
-  "profiling-ffi",
-  "profiling-replayer",
+  "crates/datadog-alloc",
+  "crates/datadog-profiling",
+  "crates/datadog-profiling-ffi",
+  "crates/datadog-profiling-replayer",
```

Comment:
This can actually be `crates/*` if we want, but there's other places that you have to add the crate, so it's not a big deal either way.